wordpress-mobile/WordPress-iOS
# Allow overriding Gutenberg dependencies with a local checkout

This PR introduces a way to run install Gutenberg from a local checkout without modifications to the Podfile.

To test:

- Running `rake dependencies:pod:install` (or any other variant that runs `pod install`) should work as before
- Running `rake dependencies:pod:install LOCAL_GUTENBERG='/path/to/gutenberg-mobile'` should install Gutenberg and its dependencies from the local copy

Update release notes:


cc @etoledom @pinarol @SergioEstevao
